protocol/contracts/libraries/Silo/LibGerminate.sol

```diff
@@ -57,6 +57,24 @@ library LibGerminate {
         int256 deltaBdv
     );
 
+    /**
+     * @notice emitted when the total germinating stalk changes. 
+     * @param germinationSeason issuance season of germinating stalk
+     * @param deltaStalk the change in the total germinating stalk.
+     * @dev the issuance season may differ from the season that this function was called in.
```

Comment:
suggest "the issuance season may differ from the season that this **event was emitted** in."

Reply:
Fixed here: https://github.com/BeanstalkFarms/Beanstalk/pull/837/commits/cdc5a0353e9298e5d74888fde498d3a9988114cc#diff-d2261445685829aad9421eae03f9644ae9d11be5713ccc0870497a0be1e58d8aR64